### PR TITLE
Doot doot this is a PR for testing things, definitely do not merge it

### PR DESCRIPTION
**Pull request by jeana-redhat**
Doot doot this is a PR for testing things, definitely do not merge it!

### CODE REVIEW

**Comment by bobfuru** [comment]
/branch/enterprise-4.10

**Review by kalexand-rh — COMMENTED** [review]
This is indeed a test PR that we should not merge.

**Review by kalexand-rh — COMMENTED** [review]
Still looks like something we can't merge. ✔️

File: release_notes/README.adoc [hunk 1]
```diff
@@ -1,4 +1 @@
-Do not add or edit release notes here. Edit release notes directly in the branch
-that they are relevant for.
-
-Release notes should be added/edited in their own PR.
+Doot doot this is a PR for testing things, definitely do not merge it!
```
Comment (michaelryanpeter) [inline release_notes/README.adoc:1]:
```suggestion
Doot doot deet this is a PR for testing things, definitely do not merge it!
```
WDYT?

**Review by michaelryanpeter — COMMENTED** [review]
testing, one two one two

**Pull request closed without merging**